### PR TITLE
ci: mark pkozlowski as available for PR reviews

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Change pullapprove configuration to re-include pkozlowski

### CODE REVIEW

**Review by atscott — APPROVED** [review]

File: .pullapprove.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ version: 3
 
 availability:
   users_unavailable:
-    - pkozlowski-opensource # OOO as of 2020-09-28
+#    - username
```
Comment (AndrewKushnir) [inline .pullapprove.yml:67]: Not sure if YAML allows having empty sections, we could consider commenting out the entire "availability" section.

**Review by alxhub — APPROVED** [review]
Reviewed-for: global-approvers

File: .pullapprove.yml (hunk 1 above)
Comment (AndrewKushnir) [reply .pullapprove.yml:67]: Checked with a few YAML formatters, having no content in a section seems to be fine.

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 509cab9972dc9ee3366649324e0b19ec9277d7f8.

**Pull request closed without merging**